browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -375,7 +375,7 @@ window.L.CalcTileLayer = window.L.CanvasTileLayer.extend({
 		}
 	},
 
-	_syncTileContainerSize: function() {
+	_syncTileContainerSize: function(force = false) {
```

Comment:
Is this `force` necessary? If we received and handled the resize to 0x0 and the size has since increased, surely `sizeChanged` will be true and this is redundant?

Reply:
I am really not happy with these comments and the review. Everyone has things to do. You definitely wasted enough time of mine. If you are not testing, then let someone else do the review.

If you are reviewing without testing, you shouldn't be learning things along the way. Enough of this step by step thing.

Either let someone else do the review or send a commit yourself, after testing it.